sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClientLiveTests.cs

```diff
@@ -22,7 +22,7 @@ public async Task GetMetricFeedbacks()
 
```

Comment:
Consider renaming the test method too

Reply:
Already done it in my next PR!

Reply:
No, I just lied. Actually I renamed the samples, not the tests.

We'll remove these tests and add better ones in one of my next two PRs, so I think it's not worth changing it.